### DISCUSSION TITLE
Add Support of fido based algorithms in sshd ( OpenSSH Server)

### DISCUSSION THREAD

**Discussion by forrest96er** [PowerShell/Win32-OpenSSH#2236] [Ideas]
Could you add support for FIDO-backed public key authentication in the OpenSSH Server (sshd)? 
This includes the following algorithms:

- sk-ssh-ed25519-cert-v01@openssh.com
- sk-ecdsa-sha2-nistp256-cert-v01@openssh.com
- sk-ssh-ed25519@openssh.com
- sk-ecdsa-sha2-nistp256@openssh.com

The implementation should be straightforward.

Please add the following configurations to sshd_config to indicate the supported implementations:

- PubkeyAuthOptions

- PubkeyAcceptedAlgorithms   
ssh-ed25519-cert-v01@openssh.com,
ecdsa-sha2-nistp256-cert-v01@openssh.com,
ecdsa-sha2-nistp384-cert-v01@openssh.com,
ecdsa-sha2-nistp521-cert-v01@openssh.com,
sk-ssh-ed25519-cert-v01@openssh.com,
sk-ecdsa-sha2-nistp256-cert-v01@openssh.com,
rsa-sha2-512-cert-v01@openssh.com,
rsa-sha2-256-cert-v01@openssh.com,
ssh-ed25519,
ecdsa-sha2-nistp256,ecdsa-sha2-nistp384,ecdsa-sha2-nistp521,
sk-ssh-ed25519@openssh.com,
sk-ecdsa-sha2-nistp256@openssh.com,
rsa-sha2-512,rsa-sha2-256

**Comment by maertendMSFT** [comment]
Issue created here: https://github.com/PowerShell/Win32-OpenSSH/issues/2237

The issue will be used to track this request.
